Tests/MongoSwiftTests/UnifiedTestRunner/UnifiedTestRunner.swift

```diff
@@ -1,8 +1,11 @@
-import MongoSwiftSync
+#if compiler(>=5.5.2) && canImport(_Concurrency)
+import MongoSwift
 import Nimble
+import NIOPosix
```

Comment:
do we need the `NIOPosix` import?

Reply:
doesn't look like this actually got removed

Reply:
This is actually needed in order to use `MultiThreadedEventLoopGroup` in this file.